Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -635,24 +645,32 @@ public void Init(IEOSCoroutineOwner coroutineOwner, string configFileName)
                 Result initResult = InitializePlatformInterface(loadedEOSConfig);
                 Debug.Log($"EOSManager::Init: InitializePlatformInterface: initResult = {initResult}");
 
-                if (initResult != Result.Success)
+#if EOS_DO_NOT_UNLOAD_SDK_ON_SHUTDOWN
+                if (initResult != Result.AlreadyConfigured)
+#endif
                 {
-#if UNITY_EDITOR
-                    ShutdownPlatformInterface();
-                    UnloadAllLibraries();
-                    ForceUnloadEOSLibrary();
-                    LoadEOSLibraries();
 
-                    var secondTryResult = InitializePlatformInterface(loadedEOSConfig);
-                    Debug.Log($"EOSManager::Init: InitializePlatformInterface: initResult = {secondTryResult}");
+                    if (initResult != Result.Success)
+                    {
+#if UNITY_EDITOR
 
-                    if (secondTryResult != Result.Success)
+#if EOS_UNLOAD_SDK_ON_SHUTDOWN
```

Comment:
It surprises me to see a region named `EOS_UNLOAD_SDK_ON_SHUTDOWN` surround a statement, and then immediately afterwards a call to `UnloadAllLibraries` is made outside it.

Reply:
That's fair; I original kept it there because `UnloadAllLibraries` doesn't actually unload all the libraries in the editor, but instead just ensures that dispose is called correctly on any of the `DLLHandles` that we have loaded, which nominally should always be called. That being said, I can make it all be inside the `EOS_UNLOAD_SDK_ON_SHUTDOWN` because that would be clearer.